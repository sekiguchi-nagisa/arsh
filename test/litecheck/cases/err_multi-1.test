#!/bin/sh
echo 1>&2 hello
echo 1>&2 world
echo 1>&2 !!
# CHECKERR: hello
# CHECKERR: world
# CHECKERR: !!!