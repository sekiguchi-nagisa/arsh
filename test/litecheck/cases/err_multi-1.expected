3c3
< !!!
---
> !!
stderr is mismatched
